src/python/foglamp/core/storage_server/storage.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Starts the FogLAMP Storage service"""
```

Comment:
I'm confused as to why we have this python code here. I was expecting the storage service to be started simply by the scheduler as at startup. What is thew benefit of these extra layers of process?